packages/components/src/interfaces/internal/ContentPageHeader.ts

```diff
@@ -42,4 +48,4 @@ export type ContentPageHeaderProps = Static<typeof ContentPageHeaderSchema> & {
   breadcrumb: BreadcrumbProps
   site: IsomerSiteProps
   LinkComponent?: LinkComponentType
-}
+} & Static<typeof imageSchemaObject>
```

Comment:
minor nit: can move this `Static` up to line 45 to be together with the other `Static`